### PR TITLE
[MIG] portal_sale_distributor: Migration to 15.0

### PR DESCRIPTION
**Pull request by JoelZilli**

### CODE REVIEW

**Comment by roboadhoc** [comment]
[Pull request status dashboard](https://runbot.adhoc.com.ar/ingadhoc/sale/pull/567).

File: portal_sale_distributor/views/product_template_views.xml [hunk 1]
```diff
@@ -39,9 +39,9 @@
         <field name="inherit_id" ref="product.product_template_tree_view"/>
         <field name="groups_id" eval="[(4, ref('group_portal_distributor'))]"/>
         <field name="arch" type="xml">
-            <field name="sequence" position="attributes">
+            <!-- <field name="sequence" position="attributes">
                 <attribute name="invisible">1</attribute>
-            </field>
+            </field> -->
```
Comment (nicomacr) [inline portal_sale_distributor/views/product_template_views.xml]: @JoelZilli remove this comment lines

**Review by nicomacr — CHANGES_REQUESTED** [review]

File: portal_sale_distributor/views/product_template_views.xml (hunk 1 above)
Comment (JoelZilli) [reply portal_sale_distributor/views/product_template_views.xml]: Done

**Comment by nicomacr** [comment]
@roboadhoc r+

**Comment by nicomacr** [comment]
@roboadhoc override=ci/oba_without_tests p=0

**Comment by nicomacr** [comment]
@roboadhoc rebase-ff

**Comment by roboadhoc** [comment]
Merge method set to rebase and fast-forward

**Pull request closed without merging**